UNRELEASED.md

```diff
@@ -6,6 +6,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+- Icon component color prop now cascades to child SVGs when using `SVGSource` as **source** prop.
```

Comment:
Should "child" be plural?

Reply:
Thanks! In French we'd pluralize both the adjective and the noun – hence my question!

Reply:
@kaelig  Happy to change if you think it's needed. English is hard. 😄

Reply:
All good – I'll follow @dpersing's guidance on this!